modules/bank/source/remote/source.go

```diff
@@ -28,19 +29,22 @@ func NewSource(source *remote.Source, bankClient banktypes.QueryClient) *Source
 }
 
 // GetBalances implements bankkeeper.Source
-func (s Source) GetBalances(addresses []string, height int64) ([]types.AccountBalance, error) {
+func (s Source) GetBalances(addresses []string, height int64) ([]types.NativeTokenAmount, error) {
 	ctx := remote.GetHeightRequestContext(s.Ctx, height)
 
-	var balances []types.AccountBalance
+	var balances []types.NativeTokenAmount
```

Comment:
Seems like a big/weird change so let's crosscheck if bad merge or same in testnet/upstream

Reply:
Same as in the upstream [chains/cheqd/mainnet](https://github.com/forbole/bdjuno/blob/chains/cheqd/mainnet/modules/bank/source/remote/source.go) branch.
But on `cosmos/v0.46.x `it's still `AccountBalance`, so I'm not 100% sure.
On testnet explorer, we used `chains/cheqd/mainnet` as the upstream branch.